partner_academic_title/models/partner_academic_title.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2015-2017 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields
+
+
+class PartnerAcademicTitle(models.Model):
+    _name = 'partner.academic.title'
+
+    name = fields.Char(required=True,
```

Comment:
Please use the same identation style here as for the field definitions in res_partner.py